toolchain/check/import_cpp.cpp

```diff
@@ -1080,6 +1080,11 @@ static auto MapBuiltinType(Context& context, SemIR::LocId loc_id,
                       context.types().GetInstId(GetSingletonType(
                           context, SemIR::BoolType::TypeInstId)));
   }
+  if (type.isCharType() && ast_context.getTypeSize(qual_type) == 8) {
```

Comment:
This will incorrectly match `signed char` and `unsigned char` as well as plain `char`. You could either use `isSpecificBuiltinType(Char_U) || isSpecificBuiltinType(Char_S)`, or use `ast_context.hasSameType(qual_type, ast_context.CharTy)` to match only plain `char` and not the other two.